seatunnel-connectors/seatunnel-connectors-spark/seatunnel-connector-spark-file/src/main/scala/org/apache/seatunnel/spark/Config.scala

```diff
@@ -23,7 +23,7 @@ object Config extends Serializable {
   final val SAVE_MODE = "save_mode"
   final val SERIALIZER = "serializer"
   final val PATH_TIME_FORMAT = "path_time_format"
-  final val DEFAULT_TIME_FORMAT = "path_time_format"
+  final val DEFAULT_TIME_FORMAT = "yyyyMMddHHmmss"
```

Comment:
DEFAULT_TIME_FORMAT should be "yyyy-MM-dd HH:mm:ss"

Reply:
OK, the official document also needs to be changed

Reply:
> OK, the official document also needs to be changed

Please modify document. Thanks

Reply:
I will submit 2 PR
PR 1 
1. modify document  spark-source-file to cancel default value format. so It will be required
2. modify document spark-sink-file,  DEFAULT_TIME_FORMAT will change to  "yyyy-MM-dd HH:mm:ss"
PR2 
fix the spark-sink-file error

Reply:
> I will submit 2 PR PR 1
> 
> 1. modify document  spark-source-file to cancel default value format. so It will be required
> 2. modify document spark-sink-file,  DEFAULT_TIME_FORMAT will change to  "yyyy-MM-dd HH:mm:ss"
>    PR2
>    fix the spark-sink-file error

Just use json as default is enough. Put there all in this PR, this good for review. Thanks

Reply:
Hi, DEFAULT_TIME_FORMAT should be "yyyyMMddHHmmss", sorry about my fault. this format should be path format, so don't need space. Soryy again.

Reply:
ok